src/javascript/ContentEditorExtensions/SelectorTypes/RichTextMarkdown/RichTextMarkdown.css

```diff
@@ -0,0 +1,8 @@
+section .sec-html ol,
```

Comment:
the filename shouldn't be `scss` ?

Reply:
it can be, but this one is just used as simple css, no dynamic styling of react component